modules/payment/src/PaymentMethodListBuilder.php

```diff
@@ -22,6 +23,16 @@ public function getFormId() {
     return 'commerce_payment_methods';
   }
 
+  protected function getEntityIds() {
```

Comment:
The parent method also has support for paging, we should recopy that code:
    // Only add the pager if a limit is specified.
    if ($this->limit) {
      $query->pager($this->limit);
    }

We also need a docblock with an inheritdoc inside.